jee-learning-platform/backend/app/models/__init__.py

```
@@ -0,0 +1,2 @@
+from .models import *
```

Ruff (0.11.9)

`from .models import *` used; unable to detect undefined names

(F403)